src/Hazelcast.Net.Testing/MultipleClusterRemoteTestBase.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Hazelcast.Clustering;
+using Hazelcast.Core;
+using Hazelcast.Testing;
+using Hazelcast.Testing.Remote;
+using NUnit.Framework;
+
+namespace Hazelcast.Testing
+{
+    internal class MultipleClusterRemoteTestBase : RemoteTestBase
```

Comment:
The multiple cluster case is completely related to blue/green staff. So, I would prefer to keep it under `Tests`. In future, in case of similar needs, we can do refactoring, I think. Also, descriptions added at https://github.com/hazelcast/hazelcast-csharp-client/pull/587/commits/8530f7a38fa9cc9e16e8fc6207ef2143104d3b01